### PR TITLE
Add new provider AutoDesk

### PR DESCRIPTION
**Pull request by pgroot91**
Q | A
-- | --
Fixed Issues? |  
Patch: Bug Fix? | No
Major: Breaking Change? | No
Minor: New Feature? | Yes

Add new provider AutoDesk

### CODE REVIEW

File: src/Provider/AutoDesk.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    public function getUserProfile()
+    {
+		$response = $this->apiRequest('userprofile/v1/users/@me');
+
+		$collection = new Data\Collection($response);
+
+		$userProfile = new User\Profile();
+
+		$userProfile->identifier = $collection->get('userId');
+		$userProfile->displayName = $collection->get('firstName') .''. $collection->get('lastName');
```
Comment (ApacheEx) [inline src/Provider/AutoDesk.php:81]: I think here you wanted to have space instead of empty string

File: src/Provider/AutoDesk.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    }
+	
+    /**
+     * {@inheritdoc}
+     *
+     * See: https://forge.autodesk.com/en/docs/oauth/v2/reference/http/users-@me-GET/
+     */
+    public function getUserProfile()
+    {
+		$response = $this->apiRequest('userprofile/v1/users/@me');
```
Comment (ApacheEx) [inline src/Provider/AutoDesk.php:74]: I think it's wrong intend - should be 4 spaces

pls fix whole file

**Review by ApacheEx — CHANGES_REQUESTED** [review]
thanks for your contribution. I see a few style issues which need to be fixed.

File: src/Provider/AutoDesk.php (hunk 1 above)
Comment (pgroot91) [reply src/Provider/AutoDesk.php]: Fixed

File: src/Provider/AutoDesk.php (hunk 2 above)
Comment (pgroot91) [reply src/Provider/AutoDesk.php:74]: Not sure why it looks like that is should have been fixed already 2 tabs. If you check view file it seems ok?

Comment (ApacheEx) [reply src/Provider/AutoDesk.php]: looks like your file has mix with tabs/spaces.

I have converted it to be only spaces and now it's good

**Review by ApacheEx — APPROVED** [review]

**Comment by ApacheEx** [comment]
thanks for it. This is good to go. Let's wait for CI and then I'll merge it to master

**Comment by pgroot91** [comment]
Fixed

**Comment by ApacheEx** [comment]
Looks good, let's release it.


Thanks!

**Pull request merged**